truss/model_inference.py

```diff
@@ -97,7 +97,7 @@ def validate_provided_parameters_with_model(
         model_class: The model class to validate against
         provided_parameters: The parameters to validate
     """
-    if type(model_class) == ClassDef:
+    if type(model_class) is ClassDef:
```

Comment:
Can `issubclass` not be used here?

Reply:
actually,  i don't think this method is even being used anywhere. I'm going to revert it.